asset/js/setup.js

```diff
@@ -32,6 +32,34 @@ function setupWithSearch(siteData) {
   VueStrap.installEvents(vm);
 }
 
+/**
+ *   site-nav-btn function to toggle itself and site navigation elements.
+ */
+// eslint-disable-next-line no-unused-vars
+function siteNavToggle(siteNavBtn) {
+  siteNavBtn.classList.toggle('morph');
+  document.getElementById('site-nav').classList.toggle('open');
+  document.getElementById('site-nav-btn-wrap').classList.toggle('open');
+}
```

Comment:
- Shall we reserve **setup.js** for Vue?
- Excessive space in <code>*&nbsp;&nbsp;&nbsp;site-nav-btn</code>.
- Attach click handlers in JavaScript instead of having to do `// eslint-disable`.